src/js/components/filterList/FilterValuesList.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import OrientationFilter from './OrientationFilter.jsx'
+import ColorFilter from './ColorFilter.jsx'
+
+export default function FilterValuesList(props) {
+    let list;
+    if(props.type === 'orientation'){
+        list = <OrientationFilter />
+    }
+    else if(props.type === 'color'){
+        list = <ColorFilter />
+    }
+    else{
+        list = props.filterValues.map((filterValue, index) => (
+            <div>
+                <label><input type={props.type} value={index}/> {filterValue.name}</label>
```

Comment:
dla props.type = 'checkbox' lepiej było by utworzyć osobny komponent, który w sobie nie zawiera <input type='checkbox' z divami onClick i zmianą wewnętrznego state. To będzie praktyczna nauka i pozwoliło by potem na wprowadzenie zmian w głównym komponencie gdzie mogłabyś przechowywać wybrane wartości filtrów